lib/list.js

```diff
@@ -0,0 +1,17 @@
+const wreck = require('wreck');
+const Boom = require('boom');
+
+module.exports = async function(email) {
+  const config = this;
+
+  if (email === 'undefined') {
+    return Boom.notFound();
+  }
+
+  const response = await wreck.get(`${config.host}/api/users/list?email=${email}`, { json: true });
+  if (!response.payload) {
```

Comment:
Verify response payload - if no users are found, the proper response would be `[]` and a `200` - not a `401`